app/models/manageiq/providers/base_manager.rb

```diff
@@ -57,7 +57,7 @@ def self.http_proxy
     end
 
     def self.default_blacklisted_event_names
```

Comment:
probably want to rename this to  `blacklisted_event_names`, since it's not a default anymore.  If we really need the default there is a way to load the settings without the user's changes, but that feels unnecessary.

Reply:
Yes this is the goal, will need to sync with a number of providers at the same time hence wip